.github/workflows/ci.yml

```diff
@@ -35,6 +35,17 @@ jobs:
       - name: Run tests
         run: FOUNDRY_PROFILE=ci forge test
 
+  test-unbounded-invariants:
```

Comment:
Why are the other profiles not being tested such as the bounded `ci`?

Reply:
Invariants default to bounded, so theyre run in the above action with the other tests